src/components/app.tsx

```diff
@@ -45,6 +46,7 @@ export default function App() {
 				>
 					<SiteContentTabs />
 				</main>
+				<UserSettings />
```

Comment:
I can't open the settings window while on the onboarding screen. I think we need to render this component also in line 26.

Reply:
Yeah, my bad! I had it there originally, and then removed it accidentally. Thanks!